outer-scripts/kubectl-kubestellar-prep_for_syncer

```diff
@@ -174,18 +183,4 @@ if ! kubectl ws "${kubectl_flags[@]}" "$mbwsname" &> /dev/null; then
     fi
 fi
 
-# Now the MBWS exists and is ready, but does it have the needed APIBinding?
-
-if ! kubectl "${kubectl_flags[@]}" get APIBinding bind-edge &> /dev/null; then
-    sleep 15
-    if ! kubectl "${kubectl_flags[@]}" get APIBinding bind-edge &> /dev/null; then
-	sleep 15
-	kubectl "${kubectl_flags[@]}" get APIBinding bind-edge > /dev/null
-    fi
-fi
```

Comment:
This code is waiting, a limited amount of time, for the mailbox controller to get its job done for this mailbox workspace. Isn't that still needed?

Reply:
For the code in the `kube-bind` branch, the mailbox-controller doesn't ensure the kcp APIBinding, right?

Reply:
right.